index.d.ts

```diff
@@ -166,7 +166,7 @@ export class Bot extends (EventEmitter as new () => TypedEmitter<BotEvents>) {
   scoreboards: { [name: string]: ScoreBoard };
   scoreboard: { [slot in DisplaySlot]: ScoreBoard };
   controlState: ControlStateStatus;
-  void;
+  void: void;
```

Comment:
Curious why void is here in the original file? Does it have a purpose in this context?

Reply:
I'm not sure if it has to do with future changes

Reply:
If so, it would be best to remove it. It can be replaced if those future changes are made.